### PR TITLE
Adds support for PHP 8.1

### LINKED ISSUE #25: PHP 8.1 support
### Feature Request

|    Q        |   A
|------------ | ------
| New Feature | yes

#### Summary

PHP with version 8.1 is released and for this component the support of the new version is still missing.

### PR DESCRIPTION
**Pull request by froschdesign**
|    Q          |   A
|-------------- | ------
| Enhancement | yes

Fixes #25

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -25,32 +25,30 @@
         }
     },
     "require": {
-        "php": "^7.3 || ~8.0.0",
-        "laminas/laminas-stdlib": "^2.7 || ^3.0",
+        "php": "^7.3 || ~8.0.0 || ~8.1.0",
```
Comment (Ocramius) [inline composer.json]: We can bump to `^7.4` here

**Review by Ocramius — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (froschdesign) [reply composer.json]: Is planned 👍

**Comment by Ocramius** [comment]
A rebase + `composer.lock` update (for the lowest PHP version) is needed here

**Review by Ocramius — APPROVED** [review]
LGTM. @froschdesign what should we do about PHPCS failures? :thinking:

**Comment by froschdesign** [comment]
@Ocramius 
> …what should we do about PHPCS failures?

I will solve the problem but no hurry, because PHP 8.1 itself is [not yet ready](https://github.com/php/doc-en/issues/980). 😜

**Comment by fezfez** [comment]
can be merge ?

**Review by Ocramius — APPROVED** [review]
Yep! 🚢 

Thanks @froschdesign!

**Comment by samsonasik** [comment]
@Ocramius should it be merged to 2.13.x branch?

**Pull request merged**